src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/PoisonTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Text.RegularExpressions;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests
+{
+    public class PoisonTests : SmokeTests
+    {
+        public PoisonTests(ITestOutputHelper outputHelper) : base(outputHelper) { }
+
+        [SkippableFact(Config.PoisonReportPathEnv, skipOnNullOrWhiteSpace: true)]
+        public void VerifyUsage()
```

Comment:
Should poison diffs result in failures or warnings?  The answer is related to how much is changing in the poison report which may be a direct result of the SdkContent changing.

Reply:
I think adding poison is probably worth failing the build but removing isn't, if that's possible.  If not I think a warning is probably appropriate right now and once we trim down the baseline some we can flip the switch.

Reply:
I agree with @crummel long term.  Short term I'm concerned this test is going to be fragile and break the build frequently.  This concern stems from the MsftToSBDiff test indicating the SB SDK files are changing fairly often.  If any of these are poisoned, then the test will fail frequently.  This is in conjunction that previously source-built files are marked as poison.

What I am leaning towards is we should enable warnings on failures to begin with, observe over time, make the appropriate adjustments, then remove the warnings on failures.

Reply:
I used the test to warn on diffs.